app/javascript/components/party/Header.js

```diff
@@ -26,7 +26,7 @@ const Header = ({ party }) => (
 </Row> */}
       <Row type="flex" justify="end" align="middle">
         <Col>
-            <a href={party.url}>{party.url}</a>
+            <a href={party.website} target="_blank">{party.website}</a>
```

Comment:
Also add the property ``rel="noopener"`` to the ``<a>`` element to avoid security issues